data.js

```diff
@@ -841,6 +841,14 @@ const emojiItems = [
         type: "movie",
         year: 1944
     },
+    {
+        title: "The Social Network",
+        emojiImgs: "💻😐🌐👤",
```

Comment:
I think it could be more clear using two or three times the emoji of a guy sitting in front of a computer ( 👨‍💻👨‍💻👨‍💻 )in combination with the emojis you picked